Source/WebKit/Platform/cocoa/XPCUtilities.h

```diff
@@ -39,6 +39,8 @@ enum class ReasonCode : uint64_t {
 void terminateWithReason(xpc_connection_t, ReasonCode, const char* reason);
 #endif
 
+#if USE(EXIT_XPC_MESSAGE_WORKAROUND)
```

Comment:
Nit: should this specify that this is related to process exit? E.g. USE(XPC_MESSAGE_FOR_PROCESS_EXIT)?